src/rachanaGJun20/CoadingExam/ATMPinValidateWithoutCollection.java

```diff
@@ -0,0 +1,79 @@
+package rachanaGJun20.CoadingExam;
+
+import java.util.Arrays;
+
+public class ATMPinValidateWithoutCollection {
+
+	// Pin number validations
+	public void validateATMPin(String[] atmpins) {
+
+		for (int index = 0; index < atmpins.length; index++) {
+			String pinnumber = atmpins[index];
+			if (pinnumber.length() == 4) {
+				if (isDigit(pinnumber)) {
+					if (isRepeatedDigit(pinnumber)) {
+						System.out.println("Number is not valid as Having repeated digits [" + pinnumber + "]");
+					} else if ((isAsc(pinnumber) || isDesc(pinnumber))) {
+						System.out.println("Number is valid  " + pinnumber);
+					} else {
+						System.out.println("Number is not valid " + pinnumber);
+					}
+				}
+			} else {
+				System.out.println("Number is not of valid length " + pinnumber);
+			}
+		}
+	}
+
+	private boolean isRepeatedDigit(String pinnumber) {
+		char[] charray = pinnumber.toCharArray();
+		for (int index = 0; index < charray.length; index++) {
```

Comment:
2 loops are not required..
first number should be verified with exactly next number.. and so on.. dont you think so ...

Reply:
Yes , Got your point......  Will change it.
Thank you.